doc/gpo/README.md

```diff
@@ -0,0 +1,26 @@
+# Group Policy Objects
+
+Since version 0.64, PowerToys is released on GitHub with GroupPolicyObject files. You can check these releases on https://github.com/microsoft/PowerToys/releases .
+
+## How to install
+
+- Copy "PowerToys.admx" to "C:\Windows\PolicyDefinitions" or equivalent path if Windows is installed in another path.
+- Copy "PowerToys.adml" to "C:\Windows\PolicyDefinitions\en-US" or equivalent path if Windows is installed in another path. If you have a different language installed, copy into your language folder as well.
```

Comment:
```suggestion
- Copy "PowerToys.adml" to "C:\Windows\PolicyDefinitions\en-US" or equivalent path if Windows is installed in another path.
```

The editor has a fallback to en-us. No need to copy english strings into both folders.